From c9fde4385a5895ab23c6615029cb3d061278f514 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 7 Mar 2019 04:13:22 -0800
Subject: [PATCH 0548/1852] dt-bindings: dma: xilinx_frmbuf: Add support for 12
 and 16 bpc RGB formats

commit 5533f87c0388538bc5f5708d8c8372bd578369a5 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds two dts strings xbgr121212 and rgb16 to select the new
12 and 16 bit RGB color formats.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
index feb00ac3c3c7..39cb6ff762c2 100644
--- a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
+++ b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
@@ -67,6 +67,8 @@ RGBA8		abgr8888	<not supported>		DRM_FORMAT_ABGR8888
 BGRA8		argb8888	<not supported>		DRM_FORMAT_ARGB8888
 BGRX8		xrgb8888	V4L2_PIX_FMT_XBGR32	DRM_FORMAT_XRGB8888
 RGBX10		xbgr2101010	V4L2_PIX_FMT_XBGR30	DRM_FORMAT_XBGR2101010
+RGBX12		xbgr2121212	V4L2_PIX_FMT_XBGR40	<not supported>
+RGBX16		rgb16		V4L2_PIX_FMT_BGR40	<not supported>
 YUV8		vuy888		V4L2_PIX_FMT_VUY24	DRM_FORMAT_VUY888
 YUVX8		xvuy8888	V4L2_PIX_FMT_XVUY32	DRM_FORMAT_XVUY8888
 YUYV8		yuyv		V4L2_PIX_FMT_YUYV	DRM_FORMAT_YUYV
-- 
2.31.1

